data/sample_generator.py

```diff
@@ -149,6 +149,7 @@ def __call__(self, sample_dict):
       - `prev_label`: An optional tensor of the same shape as `label`.
       - `next_image`: An optional next-frame tensor of the shape of `image`.
       - `next_label`: An optional next-frame tensor of the shape of `label`.
+      - `depth`: An optional tensor of thesame shape as `label`.
```

Comment:
- `depth`: An optional tensor of the same shape as `label`.
(one space between thesame)